README.md

```diff
@@ -10,6 +10,7 @@
   - [Setup](#setup)
   - [Install](#install)
 - [👥 Authors](#authors)
```

Comment:
- [ ] Please I highly recommend adding an Author section and social media links to the `README` file.

Reply:
hello how are you this required change was already implemented, I would like to know if I need another change for the authors section of my README.md file

Reply:
- [ ] Please you need to update it by adding your name `Andres` instead of `Author1`

Example:

```
👤 **Andres**
- GitHub: [@githubhandle](https://github.com/AndresEspi)
- Twitter: [@twitterhandle](https://twitter.com/Andres_Esp1nosa)
- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andres-espinosa-3bba67271/)
```

Reply:
ok perfect thank you very much for the clarification :3